.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
- [ ] Another important element when coding is the Pull Request message, as it lets other developers know our intent and our reasons for merging a piece of code to a bigger project. This message should be descriptive and meaningful and summarize the most relevant changes we have made in our feature/code.

We should update the message and describe why we want to merge these changes into the `main` branch.

Reply:
:heavy_check_mark: The updated PR message is acceptable for this project.